src/app/redux/auth/authReducer.js

```diff
@@ -0,0 +1,37 @@
+import actionTypes from './authActionTypes';
+
+const initialState = {
+  userEmail: null,
+  userRole: null,
+  error: null,
+};
+
+const authReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.AUTH_SET_ACTIVE_USER:
+      localStorage.setItem("userEmail", action.userEmail);
+      localStorage.setItem("userRole", action.userRole);
```

Comment:
We shoud keep the reducer pure as it just checks the action and returns the appropriate state